modules/cli-about-cli.adoc

```diff
@@ -3,12 +3,10 @@
 // * cli_reference/openshift_cli/getting-started.adoc
 
 [id="cli-about-cli_{context}"]
-= About the CLI
+= About the OpenShift CLI
 
-With the {product-title} command-line interface (CLI), you can create
-applications and manage {product-title} projects from a terminal. The CLI is
-ideal in situations where you:
+With the {product-title} command-line interface (CLI), you can create applications and manage {product-title} projects from a terminal. The OpenShift CLI is ideal in the following situations:
```

Comment:
IIRC, we want to refer to this as "the OpenShift CLI" as opposed to the "{product-title} CLI" (as is the case with the next sentence), correct?